src/20_Components/amp-app-banner.html

```diff
@@ -0,0 +1,91 @@
+<!---{
+    "experiments": ["amp-app-banner"],
+    "skipValidation": true
+  }--->
+<!--
+  #### Introduction
+
+[amp-app-banner](https://www.ampproject.org/docs/reference/components/amp-app-banner) provides a minimal UI for a cross-platform, fixed-position banner showing a call-to-action to install an app. 
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- #### Setup -->
+  <!--
+  Import the `amp-app-banner` component.
+  -->
+  <script async custom-element="amp-app-banner" src="https://cdn.ampproject.org/v0/amp-app-banner-0.1.js"></script>
+  <!--
+    Declare the iOS app in the meta data.
+  -->
+  <meta name="apple-itunes-app" content="app-id=828256236, app-argument=medium://p/cb7f223fad86">
+  <!--
+    The `amp-app-banner` component uses the `prefer_related_applications` key from the web app manifest. So, you could use exactly the same manifest for your actual site and get the native Chrome banner too.
```

Comment:
You're right. This absolutely makes no sense. "prefer_related_applications": true is only relevant if you prefer the native install banner over the web app one. It has no effect on amp-app-banner.